tests/integration/test_loki_configs.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+from pathlib import Path
+
+import pytest
+import yaml
+from helpers import is_loki_up, loki_config, loki_services
+from pytest_operator.plugin import OpsTest
+
+logger = logging.getLogger(__name__)
+
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+app_name = METADATA["name"]
+resources = {"loki-image": METADATA["resources"]["loki-image"]["upstream-source"]}
+
+
+@pytest.mark.abort_on_fail
+async def test_services_running(ops_test: OpsTest, loki_charm):
+    """Deploy the charm-under-test."""
+    logger.debug("deploy local charm")
+
+    await ops_test.model.deploy(
+        loki_charm, application_name=app_name, resources=resources, trust=True
+    )
+    await ops_test.model.wait_for_idle(apps=[app_name], status="active", timeout=1000)
+    assert await is_loki_up(ops_test, app_name)
+
+    services = await loki_services(ops_test, app_name)
+    assert all(status == "Running" for status in services.values()), "Not all services are running"
```

Comment:
I wonder if there's a way we could simplify this. For example `assert "Not running" not in output`.